tests/helpers/package-cache.js

```diff
@@ -309,11 +300,14 @@ PackageCache.prototype = {
    * @param {String} manifest The contents of the manifest file to write to disk.
    */
   _writeManifest: function(label, type, manifest) {
+    process.chdir(this.rootPath);
     var outputDir = quickTemp.makeOrReuse(this.dirs, label);
+    process.chdir(originalWorkingDirectory);
+
     this._conf.set(label, outputDir);
 
     var outputFile = path.join(outputDir, translate(type, 'manifest'));
-    fs.writeFileSync(outputFile, manifest);
+    fs.outputFileSync(outputFile, manifest);
```

Comment:
This started erroring on me and I looked it up. "Surely `mkdirp` + `fs.writeFileSync` exists?"